dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -510,28 +506,39 @@ public void removeAttachedFile(WorkflowTask task, String fileInode) throws DotDa
 	@WrapInTransaction
 	public void reorderAction(WorkflowAction action, int order) throws DotDataException, AlreadyExistException {
 
-		WorkflowStep step = findStep(action.getStepId());
+		final WorkflowStep step = findStep(action.getStepId());

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getStepId"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)